docs/guides/capability-sets.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Session Capability Sets For Common Testing Scenarious
```

Comment:
Maybe 'Basic Examples of Session Capability Sets'?

Reply:
changed